packages/voice/vitest.config.ts

```diff
@@ -0,0 +1,3 @@
+import defaultConfig from '../../vitest.config.js';
+
+export default defaultConfig;
```

Comment:
Maybe consider [Vitest workspace](https://vitest.dev/guide/workspace)? It has supports for extending config without having to import the base config into each file.

Example:
```ts
// vitest.workspace.ts at root
import { defineWorkspace } from 'vitest/config'

export default defineWorkspace([
  'packages/*',
  {
    extends: './vite.config.js',
  }
])
```